_posts/2020-07-21-react-native-tracker-0-1-0-released/index.md

```diff
@@ -0,0 +1,93 @@
+tags: [snowplow, react-native, tracker]
+author: Colm
+category: Releases
+permalink: /blog/2020/07/21/snowplow-react-native-tracker-0.1.0-released/
+discourse: false
+---
+
+We are pleased to announce the first production-ready release of the [Snowplow React Native Tracker](https://github.com/snowplow-incubator/snowplow-react-native-tracker), version 0.1.0. The React Native tracker imports the native Snowplow iOS and Android trackers to React Native, as Native Modules. Much of the functionality of the native versions of the tracker is available.
+
+Read on below for:
+
+1. [Key features](#features)
+2. [Quickstart guide](#quickstart)
+3. [Feedback and next steps](#feedback)
+4. [Changes since the alpha release](#alpha)
+5. [Documentation](#documentation)
+6. [Getting help](#help)
+
+<!--more-->
+
+<h2 id="features">1. Key features</h2>
+
+This first production-ready release of the Snowplow React Native tracker has the following key features, closely in line with the existing functionality of our iOS and Android trackers:
+
+- Automatic tracking: The tracker includes automatic tracking of the mobile context, screen context and session context, as well as installs and lifecycle events. All are configurable on tracker initialisation.
+
+- Standard events: Screen View, Page View, and Structured events are available.
+
+- Custom tracking: Custom Self-Describing events and contexts are available.
+
+- Setting subject data: The subject can be set using the `setSubjectData()` method.
+
+
+<h2 id="quickstart">2. Quickstart guide</h2>
+
+To install the tracker: 
+
+```
+npm install @snowplow/react-native-tracker
+```
+
+To initialize the tracker and track a screen view:
+
+```
+import Tracker from '@snowplow/react-native-tracker';
+
+initialize({
+  endpoint: 'my-endpoint.com',
+  namespace: 'namespace',
+  appId: 'my-app-id'
+});
+
+trackScreenViewEvent({screenName: 'myScreenName'})
+```
+
+For a full list of all the tracker initialization options, including automatic sessionization and lifecycle events, please see the [Snowplow React Native Tracker documentation](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/react-native-tracker/#configuration). 
+
+
+<h2 id="feedback">3. Feedback and next steps</h2>
+
+As this is an early release tracker, we welcome feedback and and of course contributions! Please feel free to open topics with ideas in our [Discourse forum](https://discourse.snowplowanalytics.com/), and add issues and bug reports to the [repository](https://github.com/snowplow-incubator/snowplow-react-native-tracker/issues).
+
+The next release will include the introduction of automated testing. PRs are still welcome, but due to the time investment required for thorough manual testing, they may not be reviewed and merged until we have those tests in place. If in doubt, open an issue or a discourse post to ask about your contribution. Important contributions, especially bugfixes, will be prioritised.
```

Comment:
Perhaps we leave this out, but link to the CONTRIBUTING.md file instead (which mentions this). Doesn't feel like we need to publicise what we didn't get round to doing.

Reply:
Good catch, have updated!